### PR TITLE
release 0.49.0

### PR DESCRIPTION
**Pull request by RenkuBot**
Release 0.49.0

This PR is auto-generated by [actions/github-script](https://github.com/actions/github-script).

/deploy

### CODE REVIEW

File: .github/workflows/renku-dev-test.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v4.1.1
-      - uses: cypress-io/github-action@v6
+      - uses: cypress-io/github-action@v5
```
Comment (leafty) [inline .github/workflows/renku-dev-test.yaml:26]: huh?

```suggestion
      - uses: cypress-io/github-action@v6
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,47 @@
 .. _changelog:
 
+0.49.0
+------
+
+The release contains bug fixes to renku core service related to project migration.
+
+This release also contains initial support for next generation 'Renku 1.0' functionality. However,
```
Comment (leafty) [inline CHANGELOG.rst:8]: Here and everywhere else:

```suggestion
This release also contains initial support for next generation 'Renku 2.0' functionality. However,
```

File: helm-chart/renku/templates/NOTES.txt [hunk 3]
```diff
@@ -9,7 +9,7 @@ can be accessed using the following one-liner (you need to have jq installed).
 kubectl get secrets -n {{ .Release.Namespace }} {{ template "renku.fullname" . }} -o json | jq -r .data.users | base64 --decode
 {{- end -}}
 
-{{ if or .Values.ui.client.privacy.banner.enabled .Values.ui.client.privacy.page.enabled -}}
+{{ if .Values.ui.client.privacy.enabled -}}
```
Comment (leafty) [inline helm-chart/renku/templates/NOTES.txt:12]: This looks like a merge conflict wrongly resolved.

**Review by leafty — CHANGES_REQUESTED** [review]
Please fix the merge conflict before releasing.

File: CHANGELOG.rst (hunk 2 above)
Comment (lokijuhy) [reply CHANGELOG.rst:8]: I will make a PR to update Renku 1.0 references for the whole changelog

**Review by leafty — APPROVED** [review]
Merge conflicts seem fixed

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**